bin/generate

```diff
@@ -186,6 +189,22 @@ function generate_hub_commit() {
   popd
 }
 
+function update_dev_branches() {
+  # Reset all dev branches to main
+
+  pushd .
+  cd /app/looker-hub
+
+  dev_branches_file="/app/lookml-generator/bin/dev_branches"
+  while read branch; do
+    git checkout $branch
+    git reset --hard main
+    git push -f origin $branch
+  done < $dev_branches_file
+
+  popd
```

Comment:
Are we using `pushd/popd` so that we can return the previous working directory?

Reply:
Yup, just a pattern I picked up from MSG.